ngraph/core/src/op/reduce_mean.cpp

```diff
@@ -68,6 +69,8 @@ bool op::v1::ReduceMean::evaluate(const HostTensorVector& outputs,
                                   const HostTensorVector& inputs) const
 {
     NGRAPH_OP_SCOPE(v1_ReduceMean_evaluate);
+    NGRAPH_CHECK(validate_host_tensor_vector(inputs, 2));
```

Comment:
Looks like we don't use the second input. Am I right?

Reply:
After including helper function to retrieve axes from host tensor, this check makes sense